randovania/games/fusion/logic_database/header.txt

```diff
@@ -111,8 +111,9 @@ Templates
 
 * Can Kill Tough Beam-Resistant Enemy:
       Any of the following:
-          Missiles ≥ 10 or Screw Attack or Can Use Power Bombs
+          Missiles ≥ 10 or Screw Attack
           Charge Beam and Have Any Beam Upgrade
+          Power Bombs ≥ 4 and Can Use Power Bombs
```

Comment:
do we need this much cushioning for PBs?

Reply:
I would prefer to have more than less cushioning. The amount of how many PBs enemies take can vary a lot, and having a little cushioning avoids situations where youre expected to go through multiple strong enemies in a row with only 1 pb.

Reply:
Fair enough, and to tell the truth, using PBs everywhere to kill tough enemies isnt really fun imo, so I'm fine with this.